### PR TITLE
docs: add documentation to item endpoints

### PR DESCRIPTION
**Pull request by pyphilia**
ref #1528

### CODE REVIEW

File: src/services/item/schemas.create.ts [hunk 1]
```diff
@@ -62,87 +54,94 @@ function itemCreateSchemaFactoryWithSettings<
...
-  Type.Object({ target: customType.UUID() }, { additionalProperties: false }),
+  customType.StrictObject({ target: customType.UUID() }),
 );
 
 export const create = {
+  operationId: 'createItem',
+  tags: ['item'],
+  summary: 'Create item',
+  description:
+    'Create item, whose possible types are folder, app, document, embeddedLink, localFile, s3File, etherpad, h5p and shortcut.',
```
Comment (spaenleh) [inline src/services/item/schemas.create.ts:141]: This could be generated from the ItemType enum ?

File: src/services/item/schemas.packed.ts [hunk 2]
```diff
@@ -36,35 +34,62 @@ export const packedItemSchemaRef = registerSchemaAsRef(
       thumbnails: Type.Optional(Type.Object({}, { additionalProperties: true })),
     },
     {
-      // Schema Options
-      additionalProperties: false,
+      description: 'Item with additional information',
```
Comment (spaenleh) [inline src/services/item/schemas.packed.ts:37]: Should we be more precise ?

File: src/services/item/schemas.ts [hunk 3]
```diff
@@ -72,168 +44,145 @@ export const settingsSchema = Type.Partial(
...
-    }),
-    '4xx': errorSchemaRef,
-  },
-} as const satisfies FastifySchema;
-
 export const reorder = {
+  operationId: 'reorderItem',
+  tags: ['item'],
+  summary: 'Reorder item',
+  description: 'Reorder item within its parent given previous item id.',
```
Comment (spaenleh) [inline src/services/item/schemas.ts:157]:
```suggestion
  description: 'Reorder item inside its parent. Place it after item denoted by the `previousItemId` parameter.',
```

File: src/services/item/schemas.ts [hunk 4]
```diff
@@ -72,168 +44,145 @@ export const settingsSchema = Type.Partial(
...
+
   params: customType.StrictObject({
-    id: customType.UUID(),
+    id: customType.UUID({ description: 'Item to reorder' }),
+  }),
+  body: customType.StrictObject({
+    previousItemId: customType.UUID({
+      description: 'Item which the item defined in params should go after',
```
Comment (spaenleh) [inline src/services/item/schemas.ts:164]: Too many `item` in this sentence.

File: src/services/item/schemas.ts [hunk 5]
```diff
@@ -72,168 +44,145 @@ export const settingsSchema = Type.Partial(
...
-        uniqueItems: true,
-      }),
-    },
-    { additionalProperties: false },
-  ),
+  operationId: 'deleteManyItems',
+  tags: ['item'],
+  summary: 'Delete many items',
+  description:
+    'Delete many items given their ids. This endpoint is asynchronous and a feedback is returned through websockets.',
```
Comment (spaenleh) [inline src/services/item/schemas.ts:178]:
```suggestion
    'Delete many items given their ids. This endpoint is asynchronous (response is given via HTTP before completion of the task) and a feedback is returned through websockets.',
```

**Review by spaenleh — APPROVED** [review]
Thank you for these improvements. I have added some comments. I know writing is hard, maybe we can find better descriptions for some of these.

**Review by ReidyT — APPROVED** [review]
Nice work! LGTM, I agree with the Basile's comments 😶‍🌫️

File: src/services/item/schemas.create.ts (hunk 1 above)
Comment (pyphilia) [reply src/services/item/schemas.create.ts:141]: Potentially we will remove it, so I let it be.

File: src/services/item/schemas.packed.ts (hunk 2 above)
Comment (pyphilia) [reply src/services/item/schemas.packed.ts:37]: I will add more details on each prop instead.

**Pull request merged**